src/attribution/onINP.ts

```diff
@@ -35,12 +35,13 @@ interface pendingEntriesGroup {
   startTime: DOMHighResTimeStamp;
   processingStart: DOMHighResTimeStamp;
   processingEnd: DOMHighResTimeStamp;
+  renderTime: DOMHighResTimeStamp;
```

Comment:
technically the code below could use `group.entries[0].startTime + group.entries[0].duration` and not add a new property here, but it seemed better to put this here as a convenience

Reply:
Yeah, new property is better I think.